beancount/parser/parser_test.py

```diff
@@ -67,6 +69,19 @@ def test_parse_doc__noerrors(self, _, __, ___):
           Assets:US:Cash             -100 USD
         """
 
+    @unittest.skipIf(sys.platform != "win32", reason="test for win32 file separator")
+    def test_win32_filename(self):
+        """filepath in document are expected to be escaped"""
+        row = r'2014-06-08 document Assets:Account1 "D:\a\beancount\beancount\path\to\document.csv"'
+
+        file = io.StringIO()
+
+        print_entries(load_string(row)[0], file=file)
+        assert (
```

Comment:
This would need to be `self.assertNotEqual()` (yes, I know that there are a couple of tests in the codebase that use plain `assert`s, these would need to be fixed too...). But I don't see why you don't compare to what you expect to get, instead than comparing to an arbitrary different result.